tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "rootDir": "./src",
     "strict": true,
     "noImplicitAny": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true,
+    "inlineSourceMap": false
```

Comment:
The `inlineSourceMap` option is set to `false`, which may not be ideal for debugging. Consider setting it to `true` if you want to enable inline source maps for easier debugging in development.